tests/client.rs

```diff
@@ -477,72 +477,72 @@ test! {
             headers: {
                 "Content-Length" => "5",
             },
-            body: (Body::wrap_stream(Body::from("hello"))),
+            body: (Body::from("hello")),
```

Comment:
This test is making sure that if the `impl Body` is stream-like, the user can specify a `content-length` header explicitly and it won't get overridden to `transfer-encoding: chunked`. So, we'll want the test. Can either comment out with a `TODO` note, or change it to use `StreamBody`.

Reply:
As with the others, I was unable to fix this. I wrote a TODO to fix it.